WalletWasabi/WabiSabi/Backend/Banning/CoinVerifier.cs

```diff
@@ -67,10 +53,17 @@ public async IAsyncEnumerable<CoinVerifyInfo> VerifyCoinsAsync(IEnumerable<Coin>
 
 		foreach (var coin in coinsToCheck)
 		{
-			if (CheckIfAlreadyVerified(coin))
+			// Step 1: Check if address is whitelisted.
+			if (Whitelist.TryGet(coin.Outpoint, out _))
+			{
+				innocentsCounter++;
+				yield return new CoinVerifyInfo(false, coin, ApiResponseItem: null, CoinVerifierReason.Whitelisted);
+			}
+			// Step 2: Check if coin is from a coinjoin.
+			else if (CoinJoinIdStore.Contains(coin.Outpoint.Hash))
 			{
 				innocentsCounter++;
-				yield return new CoinVerifyInfo(false, coin);
+				yield return new CoinVerifyInfo(false, coin, ApiResponseItem: null, CoinVerifierReason.FromCoinjoin);
```

Comment:
Can be
```suggestion
				yield return new CoinVerifyInfo(ShouldBan: false, coin, ApiResponseItem: null, CoinVerifierReason.FromCoinjoin);
```

given that we touch this bit